src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/AndroidBrokerProxy.cs

```diff
@@ -226,21 +240,48 @@ public Intent GetIntentForBrokerActivity(AuthenticationRequest request, Activity
                 // logic
                 if (intent != null)
                 {
+                    _logger.Info("BrokerProxy: Intent created from BundleResult is not null. ");
                     intent.PutExtra(BrokerConstants.BrokerRequest, BrokerConstants.BrokerRequest);
                 }
+
+                _logger.Info("BrokerProxy: Intent created from BundleResult is null. ");
             }
-            catch (AdalException)
+            catch (AdalException ex)
             {
-                throw;
+                _logger.ErrorPii(ex);
```

Comment:
do we log anything even when pii is not enabled? can potentially still log the exception type and perhaps even the message or the errorcode

Reply:
i'm pretty sure we still log it, but w/the pii info scrubbed. we do in msal. we have GetPiiScrubbedExceptionDetails, but would need to look into more. otherwise, we'd have to have double logging again, right?

---
In reply to: [286274796](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1598#discussion_r286274796) [](ancestors = 286274796)